build.plugins.toml

```diff
@@ -0,0 +1,5 @@
+[versions]
+kotlin = "1.8.10"
```

Comment:
should probably keep the comments for the various kotlin versions and why we are using this one, here.

Reply:
Added the AOSP link here. Any other comments you think would be good here let me know and will add it.

Reply:
I found the comments we had before helpful, explaining that we need to match kotlin versions of AOSP here and what Android version was using which Kotlin version, but I guess we can build this up again starting from 14.